src/Fantomas.Core/Trivia.fs

```diff
@@ -80,11 +80,17 @@ let internal collectTriviaFromCodeComments
         | CommentTrivia.LineComment r ->
             let content = source.GetContentAt r
 
-            let line = source.GetLineString(r.StartLine - 1)
+            let index = r.StartLine - 1
+
```

Comment:
Could you put these all after each other maybe?
```fsharp
let content = ...
let index = ...
let line = ...
```

Reply:
Done!